docs/usage.md

```diff
@@ -235,6 +241,20 @@ Database for taxonomic binning with kraken2 (default: none). E.g. "<ftp://ftp.cc
 Database for taxonomic classification of metagenome assembled genomes (default: none). E.g. "<tbb.bio.uu.nl/bastiaan/CAT*prepare/CAT_prepare_20190108.tar.gz>"
 The zipped file needs to contain a folder named "\_taxonomy*" and "_CAT_database_" that hold the respective files.
 
+## Assembly options
+
+### `--megahit_fix_cpu_1`
+
+Fix number of CPUs for MEGAHIT to 1. Not increased with retries (default: false). See also [Reproducibility](#reproducibility).
+
+### `--spades_fix_cpus`
+
+Fixed number of CPUs used by SPAdes. Not increased with retries (default: none).
```

Comment:
default: false ?

Reply:
Maybe, wasn't sure because in the end it is an `int` parameter.

Reply:
thinking about it, I would still opt for "none" to distinguish it from boolean parameters.

Reply:
Makes sense, agreed.